src/ontology/mondo-edit.obo

```diff
@@ -470266,7 +470267,7 @@ xref: Orphanet:972 {source="MONDO:equivalentTo"}
 xref: UMLS:C1834559 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:331775"}
 is_a: MONDO:0002254 {source="https://orcid.org/0000-0002-6601-2165"} ! syndromic disease
 is_a: MONDO:0005336 ! myopathy
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6218 {source="OMIM:160120", source="Orphanet:972"} ! KCNA1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6218 {source="Orphanet:972"} ! KCNA1
```

Comment:
the omim source should have been protected https://github.com/monarch-initiative/omim/blob/main/data/protected-disease-gene.tsv#L17